.github/workflows/ci.yml

```
@@ -186,14 +194,14 @@ jobs:
           toolchain: ${{ matrix.rust-version }}
           components: clippy
 
+      - uses: r7kamura/rust-problem-matchers@v1
+
       - uses: Swatinem/rust-cache@359a70e43a0bb8a13953b04a90f76428b4959bb6 # v2.2.0
         with:
           save-if: ${{ github.ref == 'refs/heads/master' }}
 
       - name: Run cargo clippy
-        uses: actions-rs/cargo@844f36862e911db73fe0815f00a4a2602c279505 # v1.0.3
-        with:
-          command: custom-clippy # cargo alias to allow reuse of config locally
+        run: cargo clippy --workspace --all-features --all-targets -- -A clippy::type_complexity -A clippy::pedantic -W clippy::used_underscore_binding -D warnings 
```

```suggestion
        run: cargo custom-clippy # cargo alias to allow reuse of config locally
```

Please use the alias to make sure we run the same config locally as in CI.